pkg/config/config.go

```diff
@@ -187,14 +189,16 @@ func envOverride(config *Config) error {
 	return nil
 }
 
-func ensurePortFormat(s string) string {
-	if len(s) == 0 {
-		return ""
+func ensurePortFormat(hostport string) string {
+	host, port, err := net.SplitHostPort(hostport)
+	if err != nil && strings.Contains(err.Error(), "missing port in address") {
```

Comment:
i'm not a big fan of string matching in logic, this is because there's no guarantee that the error message will always be the same. If the `net` library changed the string, we'll break. 

One way to do it is basically this: 
1. Check if the string is a number, if it is, then add a `:` to it and return. 
2. If it's not a number, then just return it as is. 

The function should be much simpler: 

```golang
func ensurePortFormat(s string) string {
  if _, err := strconv.Atoi(s); err == nil {
    return ":" + s
  }
  return s
}
```

I believe this should cover all 3 cases of `3000`, `:3000`, `<ip>:3000`

Reply:
agree. The `net` package doesn't expose the error so that would be breakage across lots of projects.

Reply:
I updated so that it uses the suggested ensurePortFormat